source/_integrations/kmtronic.markdown

```diff
@@ -0,0 +1,20 @@
+---
+title: KMTronic
+description: Integrate KMTronic devices
+ha_category:
+  - Switch
+ha_release: 0.119
+ha_codeowners:
+  - '@dgomes'
+ha_iot_class: Local Push
+ha_domain: kmtronic
+featured: false
+ha_config_flow: true
+---
+
+Integrate [KMtronic devices](https://www.kmtronic.com/) into Home Assistant.
```

Comment:
Maybe change this part into what `kmtronic` actually is? As in, an introduction to the integration.

Reply:
Thanks for the feedback!